shared/src/business/useCases/trialSessions/generateNoticeOfTrialIssuedInteractor.js

```diff
@@ -31,16 +37,30 @@ exports.generateNoticeOfTrialIssuedInteractor = async ({
   const { docketNumberWithSuffix } = caseDetail;
   const { caseCaptionExtension, caseTitle } = getCaseCaptionMeta(caseDetail);
 
+  const formattedStartDate = formatDateString(
+    trialSession.startDate,
+    FORMATS.MONTH_DAY_YEAR_WITH_DAY_OF_WEEK,
+  );
+
+  const trialStartTimeIso = createISODateString(
+    trialSession.startTime,
+    'HH:mm',
+  );
+  const formattedStartTime = formatDateString(trialStartTimeIso, 'hh:mm A');
+
+  const judgeWithTitle = await getJudgeWithTitle({
+    applicationContext,
+    judgeUserName: trialSession.judge.name,
+  });
+
   const trialInfo = {
-    address1: trialSession.address1,
-    address2: trialSession.address2,
-    city: trialSession.city,
-    courthouseName: trialSession.courthouseName,
-    judge: trialSession.judge.name,
-    postalCode: trialSession.postalCode,
-    startDate: trialSession.startDate,
-    startTime: trialSession.startTime,
-    state: trialSession.state,
+    formattedJudge: judgeWithTitle,
```

Comment:
Don't we still want to set notices for in-person proceedings too? It looks like we're updating this to provide the same trial info for either proceeding type, but maybe I'm missing something.

Reply:
In-person trial session support hasn't been added yet, because all sessions are currently remote.